Editor/CesiumCameraManagerEditor.cs

```diff
@@ -0,0 +1,59 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace CesiumForUnity
+{
+    [CustomEditor(typeof(CesiumCameraManager))]
+    public class CesiumCameraManagerEditor : Editor
+    {
+        private SerializedProperty _useMainCamera;
+        private SerializedProperty _useActiveSceneViewCameraInEditor;
+        private SerializedProperty _additionalCameras;
+
+        private void OnEnable()
+        {
+            this._useMainCamera =
+                this.serializedObject.FindProperty("_useMainCamera");
+            this._useActiveSceneViewCameraInEditor =
+                this.serializedObject.FindProperty("_useSceneViewCameraInEditor");
+            this._additionalCameras =
+                this.serializedObject.FindProperty("_additionalCameras");
+        }
+
+        public override void OnInspectorGUI()
+        {
+            this.serializedObject.Update();
+            this.DrawProperties();
```

Comment:
UI nitpick: can we use this inspector label width so that the text is fully visible before the checkbox?

```suggestion
            this.serializedObject.Update();
            EditorGUIUtility.labelWidth = CesiumEditorStyle.inspectorLabelWidth;
            this.DrawProperties();
```
This is what it will look like:

![image](https://github.com/user-attachments/assets/70ac38b2-a8a6-450a-90b7-a5264af00191)